.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,11 @@
 >^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 > Remember to delete this note before submitting your pull request.
 >
-> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
+> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
 > 
-> For pull requests on Chinese localization, set Base Branch to release-1.14
```

Comment:
Can you tag @kubernetes/sig-docs-zh-reviews team for this change next time?

Reply:
I would welcome this change under step #7 on this doc: https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/docs#prepare-the-next-docs-lead-for-success

Do you want to PR that?

/cc @daminisatya